exercism-modern.el

```diff
@@ -73,6 +73,36 @@ Defaults to first entry in $PATH, can be overridden if required."
 (defvar exercism-modern-current-track nil
   "Current track to pull exercises for.")
 
+(defcustom exercism-modern-star-icon (exercism-modern--load-from-package-root "icons/star.svg")
+  "Path to icon to use for recommended exercises."
+  :type 'file
+  :group 'exercism-modern)
+
+(defcustom exercism-modern-easy-icon (exercism-modern--load-from-package-root "icons/easy.svg")
+  "Path to icon to use for difficulty level easy exercises."
+  :type 'file
+  :group 'exercism-modern)
+
+(defcustom exercism-modern-medium-icon (exercism-modern--load-from-package-root "icons/medium.svg")
+  "Path to icon to use for difficulty level medium exercises."
+  :type 'file
+  :group 'exercism-modern)
+
+(defcustom exercism-modern-hard-icon (exercism-modern--load-from-package-root "icons/hard.svg")
+  "Path to icon to use for difficulty leve hard exercises."
+  :type 'file
+  :group 'exercism-modern)
+
+(defcustom exercism-modern-joined-icon (exercism-modern--load-from-package-root "icons/true.svg")
```

Comment:
Currently just used for joined/unjoined but I can see it being used elsewhere so I'd rather they just be generic names like success/failure

Reply:
Fair enough